index.js

```diff
@@ -62,7 +62,9 @@ exports.default = new Transformer({
       code = preprocessed.toString();
     }
 
-    const { js, css } = compile(code, options.compiler);
+    // Avoid duplicate inline CSS from <style> tags
+    const options = Object.assign({ css: false }, options.compiler);
```

Comment:
This causes an error--`options` is shadowing the `options` argument. Need to fix.

Reply:
I'm also not sure why the compiler options in `getConfig` are not being picked up here, since css is set to `false` there also. Hmm.